### DISCUSSION TITLE
Free up space

### DISCUSSION THREAD

**Discussion by jaymcd** [OpenNeptune3D/OpenNept4une#164] [General]
I just installed OpenNept4une and love it so far. That said, I have only about 750MB left on the EMMC card it seems after install. Is there any suggestions for what I can get rid of to free up room?

**Comment by halfmanbear** [comment]
After a fresh install you should have 2.2GB free.

**Reply by jaymcd** [reply]
interesting - this is what I have - 5.8GB used 746MB free. Any easy way to tell what I can get rid of to clean up?

**Reply by ml4liberty** [reply]
Did you get an answer on this? I am having a disk usage bottleneck as well.

**Reply by jaymcd** [reply]
I ended up just buying a bigger emmc card

**Reply by halfmanbear** [reply]
> Did you get an answer on this? I am having a disk usage bottleneck as well.

Commands to run from the terminal via SSH
```
sudo du -ah /var --max-depth=1 | sort -rh | head -n 20
```

Say if `home` is taking up most space then run;
```
sudo du -ah /home | sort -rh | head -n 20
```
This should show the largest directories, you can also clean up some space with apt;

```
sudo apt autoremove
sudo apt clean
```

**Comment by steve8x8** [comment]
Check the size of logs located in /var/log and in ~mks (here below printer_data IIRC), remove old ones. (I'd also add `sudo apt autoclean` in the middle of the above `sudo apt` statements.)
